src/Kronecker.jl

```diff
@@ -2,10 +2,10 @@ module Kronecker
 
 export GeneralizedKroneckerProduct, AbstractKroneckerProduct, AbstractSquareKronecker
 export AbstractKroneckerSum, KroneckerSum
```

Comment:
Can we remove these exports (and the line above), since the intention is that the user just uses the `kronecker` interface etc?